### PR TITLE
Moved the self-service import script into this repo

### PR DESCRIPTION
**Pull request by mshmoustafa**
This script currently lives here: https://gist.github.com/mshmoustafa/1024ab2b13363f5b2613f1501d6b0e99 but I'd like to move it to a more official place than my github gists.

# Changes

- Adds the "Self service receipt import script" folder
- Adds the Readme and import.js files from the gist

cc @JazmineMT

### CODE REVIEW

File: Self service receipt import script/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# A Node script that parses a CSV file of fetch tokens and imports them into [RevenueCat](https://www.revenuecat.com) using the REST API.
+**License: MIT**
+
+If you have at least 100k in monthly revenue, you can [reach out](https://www.revenuecat.com/demo/) to perform an assisted bulk receipt import.
+
+Not sure if this self-service script is right for you? [Get in touch](https://app.revenuecat.com/settings/support)!
+
+## ⚠️ Warning!
+This script is automated and performs changes to the data in RevenueCat that may be difficult to reverse. If you're not a developer, have one help you. If you are a developer, test the script on a small subset of the CSV file and check how it looks in RevenueCat before running it on the full file.
```
Comment (mshmoustafa) [inline Self service receipt import script/README.md:9]: This was missing in the original, I added this in today

**Comment by mshmoustafa** [comment]
@fire-at-will Sorry should have made the expectation clearer here but can I get a quick thumbs up (or thumbs down) whenever you have free time? Didn't mean to add this review to your plate, just wanted to increase visibility

File: Self service receipt import script/README.md (hunk 1 above)
Comment (fire-at-will) [reply Self service receipt import script/README.md:9]: Great warning 🙌

**Review by fire-at-will — APPROVED** [review]
Looks good to me!

**Pull request merged**